aether-utils/aether-client/conf/pip/requirements.txt

```diff
@@ -10,13 +10,27 @@
 #                                                                               #
 #################################################################################
 
-certifi==2018.4.16
+atomicwrites==1.2.1
+attrs==18.2.0
+certifi==2018.8.24
 chardet==3.0.4
-coverage==4.5.1
+coreapi==2.3.3
+coreschema==0.0.4
 flake8==3.5.0
-idna==2.6
+idna==2.7
+itypes==1.1.0
+Jinja2==2.10
+MarkupSafe==1.0
 mccabe==0.6.1
+more-itertools==4.3.0
+openapi-codec==1.3.2
+pluggy==0.7.1
+py==1.6.0
 pycodestyle==2.3.1
 pyflakes==1.6.0
-requests==2.18.4
-urllib3==1.22
+pytest==3.8.0
+pytest-runner==4.2
+requests==2.19.1
+six==1.11.0
+uritemplate==3.0.0
+urllib3==1.23
```

Comment:
Being `aether-client` a distributed package you don't need to upgrade dependencies, just something like `aether-common` has.
https://github.com/eHealthAfrica/aether/blob/develop/aether-common-module/conf/pip/requirements.txt

I'm just trying to reduce complexity :grimacing:

Reply:
That makes a lot of sense. Sometimes you just follow the pattern without checking to see if it's required...